pytest.ini

```diff
@@ -1,2 +1,2 @@
 [pytest]
-addopts = --cov=scripts --cov-report html --cov-report term --cov-report xml --ignore=tests/*/test*_vectors.py --junitxml=test-results.xml -vv
+addopts = --cov=scripts --cov=host_modules --cov-report html --cov-report term --cov-report xml --ignore=tests/*/test*_vectors.py --junitxml=test-results.xml -vv
```

Comment:
>host_modules [](http://example.com/codeflow?start=30&length=12)

Why need add this path? Is the default option working well?

Reply:
I have checked the coverage result, and the default setting does not cover host_modules directory.

Reply:
Could you just remove all the directories, like `--cov=scripts --cov=host_modules`?

Reply:
I have tried to remove "--cov=scripts --cov=host_modules", and pytest cannot generate coverage.xml.